### ISSUE TITLE
[Bug]: TN cannot connect to logserivce via discovery-address

### ISSUE THREAD

**Issue by aylei** [matrixorigin/matrixone#19183]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch Name

main

### Commit ID

https://github.com/matrixorigin/matrixone/commit/8bb6702f139fbab46d8bbc81decd8a49448173ad

### Other Environment Information

```Markdown
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

TN cannot connect to logservice

### Steps to Reproduce

```Markdown
start TN with hakeeper discovery address
```

**Comment by aylei** [comment]
closed via #19175

**Comment by aressu1985** [comment]
fixed
